src/Output/Checklist.php

```diff
@@ -77,6 +80,10 @@ public function updateProgressBar($update_message): void {
     $message_lines = explode(PHP_EOL, $update_message);
     foreach ($message_lines as $line) {
       if ($this->useSpinner()) {
+        if (trim($line)) {
+          $spinner->setMessage(str_repeat(' ', $this->indentLength * 2) . $line,
+            'detail');
```

Comment:
@anavarre to test this, try running `acli pull:run-scripts` locally on a Composer built site.

Reply:
@grasmash I was trying to test this out but ran into the following when doing a `cc`.

```
Parse error: syntax error, unexpected '>>' (T_SR) in /tmp/cli/src/Command/Pull/PullCommandBase.php on line 781
```